### PR TITLE
Upgrade radar_client to use radar_message library

### PR DESCRIPTION
**Pull request by bolddane**
Upgrade radar_client to remove in-place message request/response files and instead use the published radar_message library.

/cc @zendesk/zendesk-radar

### Steps to merge
 - [ ] :+1: of the team

### References
 - Jira link: https://zendesk.atlassian.net/browse/RADAR-481

### Risks
 - None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "microee": "*",
     "minilog": "*",
     "engine.io-client": "1.4.2",
-    "sfsm": "0.0.4"
+    "sfsm": "0.0.4",
+    "radar_message": "*"
```
Comment (lucasefe) [inline package.json]: I think we should set the version here, and be explicit about when it changes.

**Comment by lucasefe** [comment]
:+1:

File: dist/radar_client.js [hunk 2]
```diff
@@ -564,10 +580,14 @@ var init = function(name) {
...
 
-for(var i = 0; i < props.length; i++){
+for (var i = 0; i < props.length; i++) {
   init(props[i]);
 }
 
+Scope.prototype._buildScopePrefix = function (typeName, scope, accountName) {
+  return typeName + ':/' + accountName + '/' + scope;
+};
+
```
Comment (vanchi-zendesk) [inline dist/radar_client.js]: this build should be a separate commit on master in future.

Comment (bolddane) [inline dist/radar_client.js]: If you are referring to what appears to be *new* code in dist/radar_client.js, the truth of the matter is that for radar_client, dist/ fell behind lately, due to my efforts to limit the surface area of change in these PRs.  The best solution is to generate dist/radar_client.js on *master* only, and to build it with a script prior to publishing the latest radar_client to NPM.

**Pull request merged**